PL_2_komentarze.py

```diff
@@ -0,0 +1,2722 @@
+    parametryklienta2.paragraph_format.tab_stops.add_tab_stop(Cm(15), WD_TAB_ALIGNMENT.CENTER) # @UndefinedVariable
+    parametryklienta2.paragraph_format.left_indent = (Cm(4))
+
+    if checkifitsce() == 1:
+        parametryklienta1 = document.add_paragraph('- ciśnienie robocze PS:\t'+cisR+' bar')
+    else:
+        parametryklienta1 = document.add_paragraph('- ciśnienie robocze:\t'+cisR+' bar')
+    parametryklienta1.paragraph_format.tab_stops.add_tab_stop(Cm(15), WD_TAB_ALIGNMENT.CENTER) # @UndefinedVariable
+    parametryklienta1.paragraph_format.left_indent = (Cm(4))
+
+    if podcisn.get() != '':
+        parametryklienta5 = document.add_paragraph('- podciśnienie:\t'+pdcsn+' bar')
+        parametryklienta5.paragraph_format.tab_stops.add_tab_stop(Cm(15), WD_TAB_ALIGNMENT.CENTER) # @UndefinedVariable
+        parametryklienta5.paragraph_format.left_indent = (Cm(4))
+    else:
+        pass
+
+    if tw == 'otoczenia' or tw == 'otoczenie':
+        if checkifitsce() == 1:
+            parametryklienta3 = document.add_paragraph('- temp. wewnętrzna TS:\totoczenia')
+        else:
+            parametryklienta3 = document.add_paragraph('- temp. wewnętrzna:\totoczenia')
+    elif tw == '':
+        parametryklienta3 = document.add_paragraph('- temp. wewnętrzna:\tbrak danych')
+    else:
+        if checkifitsce() == 1:
+            parametryklienta3 = document.add_paragraph('- temp. wewnętrzna TS:\t' + tw + '°C')
+        else:
+            parametryklienta3 = document.add_paragraph('- temp. wewnętrzna:\t' + tw + '°C')
+    parametryklienta3.paragraph_format.tab_stops.add_tab_stop(Cm(15), WD_TAB_ALIGNMENT.CENTER) # @UndefinedVariable
+    parametryklienta3.paragraph_format.left_indent = (Cm(4))
+
+    if tz == 'otoczenia' or tz == 'otoczenie':
+        if checkifitsce() == 1:
+            parametryklienta4 = document.add_paragraph('- temp. zewnętrzna TS:\totoczenia')
+        else:
+            parametryklienta4 = document.add_paragraph('- temp. zewnętrzna:\totoczenia')
+    elif tz =='':
+        parametryklienta4 = document.add_paragraph('- temp. zewnętrzna:\tbrak danych')
+    else:
+        if checkifitsce() == 1:
+            parametryklienta4 = document.add_paragraph('- temp. zewnętrzna TS:\t' + tz + '°C')
+        else:
+            parametryklienta4 = document.add_paragraph('- temp. zewnętrzna:\t' + tz + '°C')
+    parametryklienta4.paragraph_format.tab_stops.add_tab_stop(Cm(15), WD_TAB_ALIGNMENT.CENTER) # @UndefinedVariable
+    parametryklienta4.paragraph_format.left_indent = (Cm(4))
+
+    if checkifitsce() == 1:
+        cisnienieproby = document.add_paragraph('- ciśnienie próby PT:\t'+ cisT + ' bar')
+        cisnienieproby.paragraph_format.tab_stops.add_tab_stop(Cm(15), WD_TAB_ALIGNMENT.CENTER) # @UndefinedVariable
+        cisnienieproby.paragraph_format.left_indent = (Cm(4))
+    else:
+        pass
+    document.add_paragraph()
+
+    czytest2 = czytest.get()
+    if czytest2 == 1:
```

Comment:
Lepiej porównywać z true/false niż z liczbami. Można uprościć ten fragment: `if czytest.get() == true` albo jeśli czytest jest wartością boolean to po prostu `if czytest`.
Jeśli `czytest` może przyjmować więcej wartości to lepiej ubrać go w Enuma i zmienić mu nazwę na np. `typTestu`. Wtedy taki enum mógłby przyjmować wartości np. `TestType.None, TestType.Hyrdaulic, TestType.Air`